### PR TITLE
scriptcs 0.17.1

### PR DESCRIPTION
**Pull request by glennblock**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/scriptcs.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Scriptcs < Formula
   desc "Tools to write and execute C#"
   homepage "https://github.com/scriptcs/scriptcs"
-  url "https://github.com/scriptcs/scriptcs/archive/v0.17.0.tar.gz"
-  sha256 "3ddf5f782d2092b2c1dd8068b2fbd97dca653ca88ade2f3aa4d764e6f2f04318"
+  url "https://github.com/scriptcs/scriptcs/archive/v0.17.1.tar.gz"
+  sha256 "e876118d82f52cbdd9569783ec9278c4ac449055aa628cdcb2d785bf8098a434"
```
Comment (fxcoudert) [inline Formula/scriptcs.rb]: The formula has `depends_on "mono" => :recommended` but it cannot build at all without `mono` (it says `mozroots: command not found`). Please turn that into a hard dependency.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/scriptcs.rb (hunk 1 above)
Comment (glennblock) [inline Formula/scriptcs.rb]: OK, no problem.

Comment (glennblock) [inline Formula/scriptcs.rb]: I have changed it to `:run` is that correct?

Comment (fxcoudert) [inline Formula/scriptcs.rb]: Just `depends_on "mono"`. And can you please squash your commits into one? With commit message `scriptcs 0.17.1`, that would be perfect! Many thanks.

Comment (glennblock) [inline Formula/scriptcs.rb]: Sure!

Comment (glennblock) [inline Formula/scriptcs.rb]: Done!

Comment (glennblock) [inline Formula/scriptcs.rb]: oops wait

Comment (glennblock) [inline Formula/scriptcs.rb]: Ok, now it should be really done.

**Review by fxcoudert — APPROVED** [review]

**Comment by glennblock** [comment]
👍

**Comment by fxcoudert** [comment]
Thanks @glennblock for your contribution!

**Pull request closed without merging**